osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -130,19 +130,24 @@ private void prepareDrawables()
             if (type == currentDrawableType)
                 return;
 
-            InternalChild = JudgementBody = new Container
+            // sub-classes might have added their own children that would be removed here if .InternalChild was used.
+            if (JudgementBody != null)
+                RemoveInternal(JudgementBody);
+
+            AddInternal(JudgementBody = new Container
             {
                 Anchor = Anchor.Centre,
                 Origin = Anchor.Centre,
                 RelativeSizeAxes = Axes.Both,
+                Depth = -float.MaxValue,
```

Comment:
i'd hope this isn't required?

Reply:
Removed as it seems to work as expected without.

Reply:
Mostly added it in an attempt to future-proof the `Remove/Add` hack. It isn't strictly required (and possibly indicates that I *should* go for `CreateBody()`).

Reply:
Arguably, `CreateBody` would add more overhead where it could potentially be avoided (for things like lighting which don't need to be refreshed on judgement type change).

I think it's fine as-is. Also, by setting `Depth` you'd potentially break usages where a derived class is trying to draw above.